src/printer/cvc/cvc_printer.cpp

```diff
@@ -942,17 +924,17 @@ void CvcPrinter::toStream(std::ostream& out,
     // Quantifiers
     case kind::FORALL:
       out << "(FORALL";
-      toStream(out, n[0], depth, false);
+      toStreamNode(out, n[0], depth, true, lbind);
```

Comment:
Just to make sure that this is not a copy&pase: Should this be really `true`? The original call was `false`.

Reply:
Yep, there were inconsistencies which I intentionally fixed here.